src/views/resetView/resetPasswordView.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { toast } from "react-toastify";
+import ResetForm from "../../components/resetPassword/resetPasswordForm";
+import resetPasswordAction from "../../actions/resetAction/resetPasswordActions";
+import ChangeForm from "../../components/resetPassword/changePasswordForm";
+import changePasswordAction from "../../actions/resetAction/changePasswordActions";
+import CircularProgressLoader from "../../commons/progressLoader";
+
+export class ResetPasswordView extends Component {
+        state = {
+          email: "",
+          password: "",
+          confirmPassword: "",
+          disabled: true,
+          errors: {
+            email: "",
+          },
+          loader: {
+            loading: false,
+          },
+        };
+    
+        componentWillReceiveProps(nextProps) {
+          if (nextProps.detail) {
+            toast.error("User associated with that email does not exist.", {
+              position: toast.POSITION.TOP_CENTER,
+              autoClose: 3000,
+              hideProgressBar: false,
+              pauseOnHover: true,
+            });
+          } else {
+            toast.success("Please check your email for the reset password link.", {
+              position: toast.POSITION.TOP_CENTER,
+              autoClose: 3000,
+              hideProgressBar: false,
+              pauseOnHover: true,
+            });
+          }
+
```

Comment:
remove this unnecessary blank line in the same block

Reply:
I have worked on this, thanks